phys2bids/cli/run.py

```diff
@@ -94,18 +98,20 @@ def _get_parser():
                           default=None)
     optional.add_argument('-ntp', '--numtps',
                           dest='num_timepoints_expected',
+                          nargs='*',
                           type=int,
-                          help='Number of expected timepoints (TRs). '
-                               'Default is 0. Note: the estimation of when the '
-                               'neuroimaging acquisition started cannot take place '
-                               'with this default.',
-                          default=0)
+                          help='Number of expected trigger timepoints (TRs). '
+                               'Default is 0. Note: the estimation of beggining of '
+                               'neuroimaging acquisition cannot take place with this default.'
+                               'Give a list of each expected ntp for multi-run recordings.',
+                          default=[0, ])
```

Comment:
Totally agreed. Once again, my fault since I was worried that the parser wouldn't have accepted a different type than lists. But it's definitely better to default them to None!

Reply:
If you just omit a `default` entirely it will be None, which should work well with the other suggestions I made about this in `phys2bids.py`!

Reply:
Cool. Let's default them to `None` though - it's clearer and easier to understand (and there's lower chances that I'm going to forget that the default lack of default is `none` :stuck_out_tongue: )!